dashboard/lib/tasks/seed.rake

```diff
@@ -97,10 +97,40 @@ namespace :seed do
   end
 
   task school_districts: :environment do
+    # use a much smaller dataset in environments that reseed data frequently.
+    school_districts_tsv = CDO.stub_school_data ? 'test/fixtures/school_districts.tsv' : 'config/school_districts.tsv'
+    expected_count = `wc -l #{school_districts_tsv}`.to_i - 1
+    raise "#{school_districts_tsv} contains no data" unless expected_count > 0
+
```

Comment:
Nice optimization! We really don't need the full data set in non-production environments.